index.d.ts

```diff
@@ -9,6 +9,13 @@ export interface Options {
 	__Only use this option if you really have to.__ Can be useful if you want to force a specific editor or implement your own auto-detection.
 	*/
 	readonly editor?: string;
+
+	/**
```

Comment:
Readme and index.d.ts docs should be in sync.

Reply:
I am sorry, but I do not see my mistake here. What have I missed?

Reply:
If you compare the two side-by-side, it's pretty clear. The description is different and the TS docs are missing the code example.